<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_form</class>
 <widget class="QMainWindow" name="add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Name_Combo_Box_Add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Quot_Line_Add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>99.99</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Min_Quot_Line_Add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>99.99</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Max_Quot_Line_Add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>99.99</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Num_contr_Line_Add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата торгов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Код фьючерса:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата погашения ценной бумаги:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущая цена фьючерса:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимальная цена фьючерса (на данных торгах):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимальная цена фьючерса (на данных торгах):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Число проданных фьючерсов:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnToMainButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Вернуться в главное меню</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddDataButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить данные</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="Torg_Date_dateEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1996</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="date">
     <date>
      <year>1996</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QDateEdit" name="Day_End_dateEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1996</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="date">
     <date>
      <year>1996</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
